code-challenges.test.js

```diff
@@ -14,6 +16,12 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// describe("functionName", () => {
+//     it("describe what my  test and function does", () => {
+//         expect(functionName("pass any params needed").toEqual("Expect output"))
+//     })
+// })
```

Comment:
I like that you placed a template at the top!  I do think there may be a small syntax error in here that may have affected your tests.